client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,324 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import SelectDropdown from '../index';
+
+describe( 'index', function() {
+	useFakeDom();
+
+	describe( 'component rendering', function() {
+		it( 'should render a list with the provided options', function() {
+			const dropdownOptions = getDropdownOptions();
+			const dropdown = mount( <SelectDropdown options={ dropdownOptions } /> );
```

Comment:
You can also extract those two lines into its own utility method to reduce some code duplication.